### PR TITLE
Channel ban

### PR DESCRIPTION
**Pull request by lleadbet**
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

## Problem/Feature

Adds the channel ban websub/eventsub topic

## Description of Changes: 

- Adds the Channel Ban Websub/Eventsub

## Checklist

- [x] My code follows the [Contribution Guide](https://github.com/twitchdev/twitch-cli/blob/master/CONTRIBUTING.md#Requirements)
- [x] I have self-reviewed the changes being requested
- [x] I have made comments on pieces of code that may be difficult to understand for other editors
- [x] I have updated the documentation (if applicable)

### CODE REVIEW

**Review by hiGarza — APPROVED** [review]
Looks great!

File: docs/event.md [hunk 1]
```diff
@@ -120,6 +123,8 @@ Allows you to test if your webserver responds to subscription requests properly.
 | `streamdown`        | Stream offline event.                                         |
 | `add-moderator`     | Channel moderator add event.                                                                               |
 | `remove-moderator`  | Channel moderator removal event.                                                                           |
+| `ban`       		  | Channel ban event.                                                                           			   |
```
Comment (BraxtonLanc43) [inline docs/event.md:126]: The pipes don't line up if we're being really strict here ;)

**Review by BraxtonLanc43 — APPROVED** [review]
Everything looks good and follows the template of other events.

File: docs/event.md (hunk 1 above)
Comment (lleadbet) [reply docs/event.md:126]: ha! i can fix that actually.

**Pull request merged**